notificationhub.go

```diff
@@ -114,7 +114,7 @@ func (h *NotificationHub) exec(ctx context.Context, method string, url *url.URL,
 	if err != nil {
 		return nil, nil, err
 	}
-	req.WithContext(ctx)
+	req = req.WithContext(ctx)
```

Comment:
I would rather use `http.NewRequestWithContext(`on line 113

Reply:
Looks like this function was introduced in [Go 1.13](https://golang.org/doc/go1.13#net/http).
Travis tests against Go 1.10, 1.11 and 1.12 which is why it's now failing.

Reply:
Good catch! Thank you